src/utilities/core/Logger.hpp

```diff
@@ -115,15 +114,22 @@ class UTILITIES_API LoggerSingleton
   SinkSetType m_sinks;
 };
 
-#if _WIN32 || _MSC_VER
+class UTILITIES_API Logger
+{
+ public:
+  Logger() = delete;
 
-/// Explicitly instantiate and export LoggerSingleton Singleton template instance
-/// so that the same instance is shared between the DLL's that link to Utilities.dll
-UTILITIES_TEMPLATE_EXT template class UTILITIES_API openstudio::Singleton<LoggerSingleton>;
```

Comment:
FWIW, openstudio::Singleton should be removed everywhere. It was needed a long time ago, but the same can be achieved with a Meyers' Singleton like you did (`static ref& instance()`)